src/store/index.js

```diff
@@ -111,10 +111,10 @@ export default createStore({
     },
   },
   actions: {
-    login({ commit }, token) {
-      const response = AuthService.login(token);
+    async login({ commit }, { token }) {
```

Comment:
You need to handle the case if the login is not successful on the backend

Reply:
Also the case when token is no longer valid should be handled, there is probably an expiration time defined on the backend